src/bootstrap/lib.rs

```diff
@@ -397,10 +399,10 @@ impl Build {
         let out = config.out.clone();
 
         let is_sudo = match env::var_os("SUDO_USER") {
```

Comment:
Can you add a comment pointing to bootstrap.py's equivalent check (and also on that check pointing to here)?